pyscriptjs/src/python/pyscript.py

```diff
@@ -256,6 +256,7 @@ def clear(self):
 
     def select(self, query, from_content=False):
         el = self.element
+        # TODO: el.content doesn't seem to exist, should it?
         if from_content:
             el = el.content
```

Comment:
The code still contains some pieces that are just dead code.
It seems like this can be safely removed, but could you please remove and then run the tests that deal with output just to be sure? Those are the `02_output` one, `zz_examples` and `repl`.
Thanks! & good found!